fix/gsi/gsiparm.anl_sd.sh

```diff
@@ -0,0 +1,92 @@
+gsi_namelist="
```

Comment:
I would suggest to combine gsiparm.anl_sd.sh into gsiparm.anl.sh

Reply:
This is doable. PM2.5 DA is just a 3DVar run with a static B generated for aerosol DA. The current design only need one parameter DO_SDDACYCLE to switch among tasks. Othwise more parameters will be needed to pass to the merged scripts.